src/applications/check-in/components/ActionItemDisplay.jsx

```diff
@@ -22,19 +22,15 @@ const ActionItemDisplay = props => {
   const dispatch = useDispatch();
   const selectApp = useMemo(makeSelectApp, []);
   const { app } = useSelector(selectApp);
-  const {
-    goToNextPage,
-    jumpToPage,
-    pages,
-    getNextPageFromRouter,
-  } = useFormRouting(router);
+  const { goToNextPage, jumpToPage, getNextPageFromRouter } = useFormRouting(
+    router,
+  );
   const nextPage = getNextPageFromRouter();
   const selectVeteranData = useMemo(makeSelectVeteranData, []);
   const { appointments } = useSelector(selectVeteranData);
 
   const displaySuccessAlert =
```

Comment:
can't this displaySuccessAlert and PreCheckInSuccessAlert component go away now?